--- /mnt/nfs/cm-14.1/hardware/qcom/bt/msm8960/libbt-vendor/src/userial_vendor.c	2018-10-05 16:28:34.564056105 +0900
+++ userial_vendor.c	2019-02-14 10:36:38.734017515 +0900
@@ -92,16 +92,16 @@
 
   while ((-1 == fd) && (retry < 7)) {
     ALOGE("init_transport: Cannot open %s: %s\n. Retry after 2 seconds",
-        bt_hci_transport_device.name, strerror(errno));
+           s_pszDevSmd[chId], strerror(errno));
     usleep(2000000);
-    fd = open(bt_hci_transport_device.name, (O_RDWR | O_NOCTTY));
+    fd = open(s_pszDevSmd[chId], (O_RDWR | O_NOCTTY));
     retry++;
   }
 
   if (-1 == fd)
   {
     ALOGE("init_transport: Cannot open %s: %s\n",
-        bt_hci_transport_device.name, strerror(errno));
+        s_pszDevSmd[chId], strerror(errno));
     return -1;
   }
 
